common/src/main/java/org/apache/uniffle/common/metrics/GRPCMetrics.java

```diff
@@ -25,6 +25,13 @@
 import io.prometheus.client.Gauge;
 
 public abstract class GRPCMetrics {
+  // Grpc server internal executor metrics
+  public static final String GRPC_SERVER_EXECUTOR_ACTIVE_THREADS_TAG = "grpcServerExecutorActiveThreads";
+  public static final String GRPC_SERVER_EXECUTOR_BLOCKING_QUEUE_SIZE_TAG = "grpcServerExecutorBlockingQueueSize";
+  private static final String GRPC_SERVER_EXECUTOR_ACTIVE_THREADS = "grpc_server_executor_active_threads";
+  private static final String GRPC_SERVER_EXECUTOR_BLOCKING_QUEUE_SIZE = "grpc_server_executor_blocking_queue_size";
+  public static final String GRCP_SERVER_CONNECTION_SIZE_TAG = "grpc_server_connection_size";
+  private static final String GRCP_SERVER_CONNECTION_SOCKETS_SIZE = "grpc_server_connection_size";
```

Comment:
Why `GRCP_SERVER_CONNECTION_SIZE_TAG` and `GRCP_SERVER_CONNECTION_SOCKETS_SIZE` have the same metrics name?

Reply:
My mistake. It should be like `public static final String GRCP_SERVER_CONNECTION_SIZE_TAG = "grpcServerConnectionSize"`.

And `GRCP_SERVER_CONNECTION_SOCKETS_SIZE` rename to `GRCP_SERVER_CONNECTION_SIZE` to keep consistent.